pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <groupId>life.qbic</groupId>
 	<artifactId>data-model-lib</artifactId>
-	<version>2.11.0</version> <!-- <<QUBE_FORCE_BUMP>> -->
+	<version>2.12.0-SNAPSHOT</version> <!-- <<QUBE_FORCE_BUMP>> -->
```

Comment:
```suggestion
	<version>2.12.0</version> <!-- <<QUBE_FORCE_BUMP>> -->
```

Reply:
Maven does this for us automatically :)

Reply:
ahh damn, I forgot that this is done for the OM ..